nucypher/utilities/sandbox/constants.py

```diff
@@ -71,3 +71,13 @@ def select_test_port() -> int:
 MOCK_URSULA_DB_FILEPATH = ':memory:'
 
 MOCK_CUSTOM_INSTALLATION_PATH = '/tmp/nucypher-tmp-test-custom-{}'.format(time.time())
+
+MOCK_REGISTRY_FILEPATH = '/tmp/nucypher-tmp-mock-registry.json'
```

Comment:
Perhaps, makes sense to make a non-permanent name (like the one including time), so that it doesn't overlap with something else by accident?